### PR TITLE
Update the Solana documentation

### PR DESCRIPTION
**Pull request by dwightjl**
Update the Solana documentation to use the new code samples

### CODE REVIEW

**Comment by dwightjl** [comment]
Rebased to main

File: docs/Solana/DataFeeds/using-data-feeds-solana.md [hunk 1]
```diff
@@ -45,6 +44,8 @@ Before you begin, set up your environment for development on Solana:
     node --version
```
Comment (aelmanaa) [inline docs/Solana/DataFeeds/using-data-feeds-solana.md:44]: Same comment as above: Node version should be 14 or higher

File: docs/Solana/overview.md [hunk 2]
```diff
@@ -36,17 +36,17 @@ The examples in the Chainlink documentation use the following languages, tools,
 - [Rust](https://docs.solana.com/developing/on-chain-programs/developing-rust): A general-purpose programming language designed for performance and memory safety
 - [Anchor](https://project-serum.github.io/anchor/getting-started/introduction.html): A framework for the [Solana Sealevel runtime](https://github.com/solana-labs/sealevel) that provides several developer tools
 - [Solana CLI](https://docs.solana.com/cli): The Solana command line interface
-- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-- [Node.js 12 or higher](https://nodejs.org/en/download/)
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git): Used to clone the example code repository
+- [Node.js 12 or higher](https://nodejs.org/en/download/): Used to run client code
```
Comment (aelmanaa) [inline docs/Solana/overview.md:40]:
```suggestion
- [Node.js 14 or higher](https://nodejs.org/en/download/): Used to run client code
```

**Review by aelmanaa — DISMISSED** [review]

File: docs/Solana/DataFeeds/using-data-feeds-off-chain.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+metadata:
+  title: "Using Data Feeds Off-Chain (Solana)"
+  description: "How to use Chainlink Data Feeds in your off-chain applications."
+---
+
+Chainlink Data Feeds are the quickest way to access market prices for real-world assets. This guide demonstrates how to read Chainlink Data Feeds on the Solana Devnet using off-chain examples in the [Chainlink Solana Starter Kit](https://github.com/smartcontractkit/solana-starter-kit). To learn how to use Data Feeds in your on-chain Solana programs, see the [Using Data Feeds On-Chain](/docs/solana/using-data-feeds-solana/) guide.
+
+To get the full list of Chainlink Data Feeds on Solana, see the [Solana Feeds](/docs/solana/data-feeds-solana/) page.
+
+{% include 'data-quality.md' %}
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply docs/Solana/DataFeeds/using-data-feeds-off-chain.md:17]: Mainnet feeds are added in a separate PR here: https://documentation-git-fork-dwightjl-dwightjl-s-da7ce4-chainlinklabs.vercel.app/docs/solana/data-feeds-solana/

File: docs/Solana/DataFeeds/using-data-feeds-solana.md (hunk 1 above)
Comment (dwightjl) [reply docs/Solana/DataFeeds/using-data-feeds-solana.md:44]: Done.

**Comment by dwightjl** [comment]
Squashed and rebased to main.

**Review by ritika-b — APPROVED** [review]
LGTM

**Pull request merged**